ebpfapi/rpc_client.cpp

```diff
@@ -1,24 +1,27 @@
 // Copyright (c) Microsoft Corporation
 // SPDX-License-Identifier: MIT
 
+#include <ctype.h>
+#include <iostream>
+#include <stdio.h>
+#include <stdlib.h>
+#include <windows.h>
 #include "ebpf_api.h"
 #pragma warning(push)
 #pragma warning(disable : 4100) // 'identifier' : unreferenced formal parameter
 #pragma warning(disable : 4244) // 'conversion' conversion from 'type1' to
                                 // 'type2', possible loss of data
 #undef VOID
-#include "ebpf_verifier.hpp"
 #pragma warning(pop)
 #include "ebpf_windows.h"
-#include "header.h"
 #include "rpc_interface_c.c"
 
 #pragma comment(lib, "Rpcrt4.lib")
 
 static RPC_WSTR _string_binding = nullptr;
-static const WCHAR* _protocol_sequence = L"ncacn_np";
+static const WCHAR* _protocol_sequence = L"ncalrpc";
 
-#define RPC_SERVER_ENDPOINT L"\\pipe\\ebpf_service"
+#define RPC_SERVER_ENDPOINT L"ebpfsvc rpc server"
```

Comment:
Don't use static endpoints. Use a dynamic endpoint to avoid spoofing.

Reply:
Changed to dynamic endpoints.